Dockerfile

```diff
@@ -1,16 +1,17 @@
 ARG PYTHON_VERSION=3.8
 FROM python:${PYTHON_VERSION}-slim
 
+RUN apt-get update && apt-get install -y build-essential autoconf \
+	&& pip install -U contrast-agent
```

Comment:
Could pip installing the agent happen in its own RUN line? often easier for debugging

Reply:
done and merging